validation_utils.test.js

```diff
@@ -0,0 +1,113 @@
+const test = require('ava');
+const { check, gen, property } = require('testcheck');
+
+var ValidationUtils = require("./validation_utils.js");
```

Comment:
nit: single quotes for string.

Reply:
@fitzn maybe a followup could be to install a code sniffer on circle, i'm not really across what should be single and double quoted and it's a lot of work to review that kind of thing by hand

Reply:
Yeah - I don't feel strongly one way or the other, just pointed out for consistency with the other strings in the repo, but after checking again, there are actually instances of both. 🤷‍♂️